code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -176,16 +176,14 @@ stare
 		for(var/turf/simulated/floor/feather/T in view(max_dist, holder.owner))
 			return FALSE
 
-
 /datum/aiTask/sequence/goalbased/nest/get_targets()
 	. = list()
 	var/mob/living/critter/flock/F = holder.owner
-
 	for(var/turf/simulated/floor/T in view(max_dist, holder.owner))
-		if(!isfeathertile(T))
-			if(F?.flock && !F.flock.isTurfFree(T, F.real_name))
-				continue
-			. += T
+		if(F?.flock && !F.flock.isTurfFree(T, F.real_name))
+			continue
+		. += T
+		break
```

Comment:
Wasn't the not flocktile check based on the conversion part of the task?

Reply:
the precondition already fails on any flock tiles within a 2 tile radius. It can be assumed that any turfs discovered by get_targets are some non-flock turf

```dm
		for(var/turf/simulated/floor/feather/T in view(max_dist, holder.owner))
			return FALSE
```